src/simpleNbox-runtime.cpp

```diff
@@ -244,20 +244,16 @@ void SimpleNbox::stashCValues( double t, const double c[] )
     const fluxpool npp_total = sum_npp();
     const fluxpool rh_total = sum_rh();
 
-    // TODO: If/when we implement fire, update this calculation to include it
-    // (as a negative term).
     // BBL-TODO this is really "exchange" not a flux
```

Comment:
When are these TODOs going to be addressed? Is there a github issue related to them?

Reply:
I don't think we have a GitHub issue but it'll be related to dealing with #397 . I will add as an issue too though.